workspace.json

```diff
@@ -27,14 +27,17 @@
           }
         },
         "build": {
-          "builder": "@nrwl/web:bundle",
+          "builder": "@nrwl/workspace:run-commands",
```

Comment:
So if I understood it correctly – this is already supported https://github.com/nrwl/nx/pull/2226? If so I would just create issue to refactor it later. I think the current solution is good enough for us now. What do you think?

Reply:
yup!